cmd/hypervisor/commands/root.go

```diff
@@ -117,7 +121,25 @@ func prepareConfig(args []string) (conf hypervisor.Config) {
 	return conf
 }
 
-func prepareMockData(hv *hypervisor.Hypervisor) {
+func prepareDmsg(conf hypervisor.Config) *dmsg.Client {
+	dmsgC := dmsg.NewClient(conf.PK, conf.SK, disc.NewHTTP(conf.DmsgDiscovery), dmsg.DefaultConfig())
+	go dmsgC.Serve()
+
+	<-dmsgC.Ready()
+	return dmsgC
+}
+
+func serveDmsg(ctx context.Context, hv *hypervisor.Hypervisor, conf hypervisor.Config) {
+	go func() {
+		if err := hv.ServeRPC(ctx, conf.DmsgPort); err != nil {
+			log.WithError(err).Fatal("Failed to serve RPC client over dmsg.")
+		}
+	}()
+	log.WithField("addr", fmt.Sprintf("dmsg://%s:%d", conf.PK, conf.DmsgPort)).
```

Comment:
What do you think about this instead?
```suggestion
	log.WithField("addr", dmsg.Addr{PK: conf.PK, Port: conf.DmsgPort}).
```